### ISSUE TITLE
Spent time - Spent time should be counted by the spent_at column, not by created_at

### ISSUE THREAD

**Issue by danielciobanica** [zubroide/gitpab#63]
We can add time spent on gitlab Today(2020-08-28), for last week (e.g. /spend 4h 2020-08-20).

Reports will count the hours by the created_at date which is Today (2020-08-28)

I think we should change this because the and also update the filter to search for spent_at time.

I will push the changes

**Comment by mnvx** [comment]
@danielciobanica Thanks for involve, please read similar issue #36

**Comment by danielciobanica** [comment]
ok, i guess we can close this issue. Btw, i thought it's an open source repository. Is there any chance to push my code?

**Comment by mnvx** [comment]
You are may fork of the project and create pull request

**Comment by danielciobanica** [comment]
ok, i'll do that

**Comment by KeldNielsen86** [comment]
I am following this, as I have same feature request :).. thank you   -> still using "created_at" time and not "spend_at" time

**Comment by danielciobanica** [comment]
https://github.com/danielciobanica/gitpab

You can clone the project from here. I forked it and pushed my changes to use the spend_at time

**Comment by KeldNielsen86** [comment]
> https://github.com/danielciobanica/gitpab
> 
> @KeldNielsen86 - You can clone the project from here. I forked it and pushed my changes to use the spend_at time

Hi @danielciobanica 

what else is the difference between your projects?.  (I prefer just to update this project) as i had many problems installing this project on my server.)

isen 't it possible to help @mnvx in order to change this in this project? :)


thank you 

regards
Keld Nielsen

**Comment by danielciobanica** [comment]
@KeldNielsen86 - nothing else is different.

I created a pull-request in the past but it wasn't approved. You can read why here: 

https://github.com/zubroide/gitpab/pull/64

@mnvx calculates employees salaries based on the time spent and he wants to make sure employee can't add time spent in the past

**Comment by KeldNielsen86** [comment]
Ahh. I see.  it makes sense. (good point).   If my developers register time which in the previous payment period, they will lose those hours.  ( else I like the idea that they are able to register hours spend for instance yesterday, in case they forgot.  

thank you for the answer

**Comment by danielciobanica** [comment]
Yes, it depends on what are you using this time report for. But everybody may forget at some point to register the time spent, so that's why i needed the report based on spent_at instead of created_at

**Comment by KeldNielsen86** [comment]
@danielciobanica Hi again, now i finally got time to "upgrade" my time registration and I did download your latest fixes, however the 

/spend 30m <date in the past>  command stil show me "created at" date in my gitpab. do i need to do anything else than 
- git pull
- npm run prod
- npm install

regards
